Makefile

```diff
@@ -200,7 +357,8 @@ test:
 
 tox:
 	coverage erase
-	tox
+	tox -e py27
```

Comment:
I am not sure, but would this not mean that only py27 environment is tested when you run `make tox` ?? There is more environments in there.